pkg/sql/parser/builtins.go

```diff
@@ -1101,37 +1222,41 @@ var Builtins = map[string][]Builtin{
 	"radians": {
 		floatBuiltin1(func(x float64) (Datum, error) {
 			return NewDFloat(DFloat(x * math.Pi / 180.0)), nil
-		}, ""),
+		}, "Converts `val` as a degree value to a radians value."),
 	},
 
 	"round": {
 		floatBuiltin1(func(x float64) (Datum, error) {
 			return round(x, 0)
-		}, ""),
+		}, "Rounds `val` to the nearest integer."),
 		decimalBuiltin1(func(x *inf.Dec) (Datum, error) {
 			return roundDecimal(x, 0)
-		}, ""),
+		}, "Rounds `val` to the nearest integer."),
 		Builtin{
-			Types:      ArgTypes{TypeFloat, TypeInt},
+			Types:      NamedArgTypes{{"input", TypeFloat}, {"decimcal_accuracy", TypeInt}},
```

Comment:
Should `decimcal_accuracy` be `decimal_accuracy`? If so, this also needs to be updated in the description.